aashish201098/README.md

```diff
@@ -23,3 +23,7 @@
 - [ ] Image Processing
 
 ## The Timeline
+
+### Day 0: January 1, 2020
+
+**Today's Progress**: Solved questions on Codeforces
```

Comment:
It would be cool if you could add links to the problem and your solution in the readme :)
Even without it, it is okay to merge. Let me know :+1:

Reply:
Merge it this time. I will include it next time.

Reply:
cool